frontend/src/redux/museumArtworkActions.js

```diff
@@ -0,0 +1,88 @@
+const getIIIFLevel = (artwork, displayWidth) => {
+  return {
+    url: 'https://www.artic.edu/iiif/2/' + artwork.image_id + '/full/' + displayWidth + ',/0/default.jpg',
+    width: displayWidth,
+    height: Math.floor(artwork.thumbnail.height *
+      displayWidth / artwork.thumbnail.width),
+  };
+};
+
+const getArtworkInfo = (artwork) => {
+  const alt = artwork.thumbnail.alt_text;
+  const linkToImage = getIIIFLevel(artwork, 500);
+  const title = artwork.title;
+  const artworkInfo = new Map();
+  artworkInfo.set('alt', alt);
+  artworkInfo.set('url', linkToImage.url);
+  artworkInfo.set('title', title);
+  return artworkInfo;
+};
+
+function getMuseumArtworks(param) {
+  const API = 'https://aggregator-data.artic.edu/api/v1/'+param;
+  return fetch(API, {
+    headers: {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    },
+  })
+      .then(handleErrors)
+      .then((res) => res.json());
+}
+
+function filterArtworks(artworks) {
+  const artworksArr = [];
+  for (const artwork of artworks) {
+    if (artwork.thumbnail != null) {
+      artworksArr.push(getArtworkInfo(artwork));
+    }
+  }
+  return artworksArr;
+}
+
+export function fetchMuseumArtworks() {
+  return (dispatch) => {
+    dispatch(fetchMuseumArtworksBegin());
+    return getMuseumArtworks('artworks?limit=9')
```

Comment:
Why a limit of 9? Magic numbers like this should often times be stored in a constant so that the name of the constant is more self-documenting.

Reply:
In the next PR, 9 gets replaced with an argument that is passed to fetchMuseumArtworks() but will keep this in mind going forward.